### ISSUE TITLE
Vers. 1.0 - compilation with gcc 4.9.2 (@Fedora 21) crashes :  unregognized option '-Wp' !

### ISSUE THREAD

**Issue by sintemalen** [nullsatz/gputools#12]
make

/usr/local/cuda/bin/nvcc -c -Xcompiler "-fpic -I/usr/local/include -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic" -I. -I"/usr/local/cuda/include" -I"/usr/lib64/R/include" rinterface.cu -o rinterface.o
gcc: error: unrecognized command line option ‘-Wp’
Makefile:32: recipe for target 'rinterface.o' failed

Any hint ?

**Comment by czarrar** [comment]
I got the same issue. The resolution was to put a backslash on the comma after the -Wp. However, I don't know how to make this change in the configure file.

**Comment by sintemalen** [comment]
---- On Tue, 08 Mar 2016 20:49:33 +0100 Zarrar Shehzad  wrote ---- 
>I got the same issue. The resolution was to put a backslash on the comma after the -Wp. 

Thanks a lot !
Modified 'Makefile' (line: "CFLAGS="), accordingly, which helped to compile 'gputools.so'

> However, I don't know how to make this change in the configure file.

Me, neither.

Best,
Frank

**Comment by czarrar** [comment]
Yes, that's what I did too. I first tried to install the package. The installer made the Makefile, which I modified. Then I ran 'make' in the 'src' folder. And then finally tried to install the package again.

**Comment by jvzl** [comment]
I'm having the same issue. To which value am I supposed to modify CFLAGS on 'Makefile'?

**Comment by sintemalen** [comment]
Modify line 'CFLAGS :  put '\' after 'Wp' & before ',' 

CFLAGS = -O2 -g -pipe -Wall -Werror=format-security -Wp\,-D_FORTIFY_SOURCE=2 -fexceptions -fstack-protector-strong --param=ssp-buffer-size=4 -grecord-gcc-switches -m64 -mtune=generic

Best,

Frank

---- On Sun, 19 Jun 2016 12:03:06 +0200 jvzl  wrote ----
